src/AzureSqlActionHelper.ts

```diff
@@ -146,16 +166,50 @@ export default class AzureSqlActionHelper {
         else if (maximumVersion === sqlPackagePatInstalledWithSSDT[1]) {
             sqlPackagePath = sqlPackagePatInstalledWithSSDT[0];
         }
+        
 
-        if (!sqlPackagePath) {
-            throw new Error('Unable to find the location of SqlPackage.exe');
+        if (sqlPackagePath == '') {
```

Comment:
I think this will work anyway, but to prevent future regressions, may want to do `===` if this logic changes in the future